client/state/rewind/size/test/reducer.js

```diff
@@ -82,4 +82,30 @@ describe( 'rewind.size reducers', () => {
 			).toEqual( 7 );
 		} );
 	} );
+	describe( 'backupsStopped', () => {
+		it( 'should return false when there is nothing in the state', () => {
+			expect(
+				sizeReducer( undefined, { type: REWIND_SIZE_SET, size: {} } ).backupsStopped
+			).toEqual( false );
+		} );
+		it( 'should return the value when the action is REWIND_SIZE_SET', () => {
+			expect(
+				sizeReducer( undefined, { type: REWIND_SIZE_SET, size: { backupsStopped: true } } )
+					.backupsStopped
+			).toEqual( true );
+		} );
+		it( 'should return new the value when it is updated and the action is REWIND_SIZE_SET', () => {
+			expect(
+				sizeReducer(
+					{ backupsStopped: true },
+					{ type: REWIND_SIZE_SET, size: { backupsStopped: false } }
+				).backupsStopped
+			).toEqual( false );
+		} );
+		it( 'should return null when the action is not REWIND_SIZE_SET', () => {
```

Comment:
I'd suggest to rephrase this description to `should return null when the action is not REWIND_SIZE_SET and the state is undefined`.

In addition, based on this test case, should we also test that it returns the `backupsStopped` value from state when `REWIND_SIZE_GET` action is dispatched?

Reply:
> In addition, based on this test case, should we also test that it returns the backupsStopped value from state when REWIND_SIZE_GET action is dispatched?

The `REWIND_SIZE_GET` action doesn't exist, wondering why we have added in other tests, I will remove them.